ui/app/AppLayouts/Profile/views/ContactsView.qml

```diff
@@ -114,16 +119,15 @@ SettingsContentBase {
                 id: pendingRequestsBtn
                 objectName: "ContactsView_PendingRequest_Button"
                 width: implicitWidth
-                enabled: root.contactsStore.receivedContactRequestsModel.count > 0 ||
-                         root.contactsStore.sentContactRequestsModel.count > 0
+                enabled: receivedRequests.count > 0 || sentRequests.count > 0
```

Comment:
Hmm, where are these `receivedRequests` and `sentRequests` coming from?

Reply:
(same below in this file)

Reply:
It was coming from `ContactsListPanel` but was wrong anyways. Already fixed.